### PR TITLE
Restored missing storage of auth state after login.

### PR DESCRIPTION
**Pull request by anastasiiaiurok-okta**
I've found out that auth state storage was broken after logout change.
Fixed that.

### CODE REVIEW

File: Okta/OktaAuth.swift [hunk 1]
```diff
@@ -47,6 +47,8 @@ public struct OktaAuthorization {
 
                         // Set the local cache and write to storage
                         OktaAuth.tokens = tokenManager
+                        self.storeAuthState(tokenManager)
```
Comment (IldarAbdullin-okta) [inline Okta/OktaAuth.swift]: storeAuthState function already has that call - OktaAuth.tokens = tokenManager. Delete line 49?

Comment (anastasiiaiurok-okta) [reply Okta/OktaAuth.swift]: Fixed.

File: Okta/OktaAuth.swift [hunk 2]
```diff
@@ -46,7 +46,8 @@ public struct OktaAuthorization {
                         let tokenManager = try OktaTokenManager(authState: authResponse, config: config)
 
                         // Set the local cache and write to storage
-                        OktaAuth.tokens = tokenManager
+                        self.storeAuthState(tokenManager)
+
```
Comment (jmelberg-okta) [inline Okta/OktaAuth.swift]: Is this an intended extra space?

**Comment by IldarAbdullin-okta** [comment]
> @anastasiiaiurok-okta - Because a previous change resulted in a regression, can you please add a test to account for caching the authentication state?

UI tests target has coverage for reading from key chain. So I think we can merge this pull request

**Pull request merged**